components/EmotionForm.js

```diff
@@ -46,95 +45,198 @@ export default function EmotionForm({ onSubmit, defaultValue }) {
 
   return (
     <>
-      <StyledSubheadline>
-        {defaultValue ? "Update your Emotion:" : "Add your Emotion:"}
-      </StyledSubheadline>
-      <StyledEmotionForm onSubmit={handleSubmit}>
-        <label htmlFor="emotion" $hasError={hasError}>
-          Emotion (type)*
-        </label>
-        <select
-          defaultValue={defaultValue?.emotion || ""}
-          id="emotion"
-          name="emotion"
-        >
-          <option value="">---Choose an emotion---</option>
-          {emotions.map((emotion, id) => (
-            <option key={id} value={emotion}>
-              {emotion}
-            </option>
-          ))}
-        </select>
-
-        <label htmlFor="intensity">Emotion intensity*</label>
-        <input
-          id="intensity"
-          name="intensity"
-          value={selectedIntensity}
-          onChange={(event) => {
-            setSelectedIntensity(event.target.value);
-          }}
-          type="range"
-          min="1"
-          max="10"
-          step="1"
-        />
-        <p>{selectedIntensity}</p>
-
-        <label htmlFor="date-time" $hasError={!defaultValue?.dateTime}>
-          Date and Time*
-        </label>
-        <input
-          id="date-time"
-          name="dateTime"
-          type="datetime-local"
-          defaultValue={defaultValue?.dateTime || currentDateTime}
-        />
-
-        <label htmlFor="notes">Notes</label>
-        <textarea
-          id="notes"
-          name="notes"
-          defaultValue={defaultValue?.notes || ""}
-          placeholder="Please describe your feelings"
-          maxLength="150"
-        ></textarea>
-
-        <button type="submit">{defaultValue ? "Save" : "Submit"}</button>
-
-        {formError && <StyledError>{formError}</StyledError>}
-
-        {successMessage && <StyledSuccess>{successMessage}</StyledSuccess>}
-      </StyledEmotionForm>
+      <StyledFormContainer>
+        <StyledSubheadline>
+          {defaultValue ? "Update your Emotion:" : "Add your Emotion:"}
+        </StyledSubheadline>
+        <StyledEmotionForm onSubmit={handleSubmit}>
+          <label htmlFor="emotion">Emotion (type)*</label>
+          <SelectEmotionContainer>
+            <StyledSelectEmotion
+              defaultValue={defaultValue?.emotion || ""}
+              id="emotion"
+              name="emotion"
+            >
+              <option value="">---Choose an emotion---</option>
+              {emotions.map((emotion, index) => (
+                <option key={index} value={emotion}>
```

Comment:
here is a very typical mistake: you are using the index as a key. Please remember this: Never use the index of an array as the key. Please use emotion instead or give your emotions ids.